src/reference/01-General-Info/90-Changes/45-Migrating-from-sbt-0.12.x.md

```diff
@@ -2,19 +2,26 @@
 out: Migrating-from-sbt-012x.html
 ---
 
+  [Organizing-Build]: Organizing-Build.html
+
 ## Migrating from sbt 0.12.x
 
 ### Introduction
 
-Before sbt 0.13 (sbt 0.9 to 0.12) it was very common to see in builds the usage of two aspects of sbt:
+Before sbt 0.13 (sbt 0.9 to 0.12) it was very common to see in builds the usage of three aspects of sbt:
 
 * the key dependency operators: `<<=`, `<+=`, `<++=`
 * the tuple enrichments (apply and map) for TaskKey's and SettingKey's (eg. `(foo, bar) map { (f, b) => ... }`)
```

Comment:
For new users, reducing duplicate concepts in the DSL would reduce the cognitive load. Maybe tuple enhancement is a good candidate for a bridging plugin to aid the old users.